e2e/iam/orgs_test.go

```diff
@@ -46,28 +39,21 @@ func TestOrgs(t *testing.T) {
 		cmd := exec.Command(cliPath, iamEntity, orgEntity, "ls")
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		a := assert.New(t)
+		if a.NoError(err, resp) {
+			var orgs mongodbatlas.Organizations
+			if err := json.Unmarshal(resp, &orgs); err != nil {
+				t.Fatalf("unexpected error: %v", err)
+			}
+			a.NotEmpty(orgs.Results)
+			orgID = orgs.Results[0].ID
 		}
```

Comment:
I really like this way of doing it 👏

Reply:
you can thank @robcarlan-mlab for adding `assert` to our libs :)